events.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
```

Comment:
Should this file be placed under src/main/webapp? Note that we also need a PR to add the pom.xml files and other associated template code.